File: app/src/main/java/com/tanner/study/ui/a_view/fragment/Practice07Fragment.java
package com.tanner.study.ui.a_view.fragment;

import android.os.Bundle;
import android.support.v4.app.Fragment;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import com.tanner.study.R;

import butterknife.ButterKnife;

public class Practice07Fragment extends Fragment {


    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
    }

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {
        View view = inflater.inflate(R.layout.fragment_practice07, container, false);
        ButterKnife.bind(this, view);
        return view;
    }

}

File: app/src/main/java/com/tanner/study/ui/a_view/f_anim/fragment/Practice07Fragment.kt
package com.tanner.study.ui.a_view.f_anim.fragment

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import com.tanner.study.R
import com.tanner.study.base.BaseFragment

/**
 * Created by Tanner on 2017/9/13.
 */
class Practice07Fragment : BaseFragment(){
    override val title: String?
        get() = "Interpolator"

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
    }

    override fun onCreateView(inflater: LayoutInflater?, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        return inflater?.inflate(R.layout.aview_fanim_practice07, container, false)

    }
}